### DISCUSSION TITLE
Limitations?

### DISCUSSION THREAD

**Discussion by Lemodo20** [abuzuhri/Amazon-SP-API-CSharp#506] [General]
Hi,

since a few days i use your API in some Practice test. last week it run more or less wel and need about 200-300 seconds to get from Amazon an Answer on POST_FLAT_FILE_CONVERGENCE_LISTINGS_DATA -Requests. But since yesterday i wait endless. Even if i restart it, i don't get an final answer from Amazon. Could it be, that my quata is full? How can i find that out?

**Comment by abuzuhri** [comment]
No its not related to quote its amazon from time to time the API getting very slow and then they solve it
